src/ui/components/shared/LoadingTip.tsx

```diff
@@ -1,61 +1,63 @@
-import React, { useEffect, useRef } from "react";
-import { useDispatch, useSelector } from "react-redux";
-import { setLoadingPageTipIndex } from "ui/actions/app";
-import { selectors } from "ui/reducers";
+import React, { FC, useEffect, useRef, useState } from "react";
+import CloudIcon from "../../../image/zypsy/zypsy-security-icon1.svg";
+import KeyIcon from "../../../image/zypsy/zypsy-security-icon2.svg";
+import ShieldIcon from "../../../image/zypsy/zypsy-security-icon3.svg";
 
 const TIPS = [
   {
     title: "SAML 2.0 / SSO Authentication",
     description:
       "Replay supports single sign-on via Google SAML 2.0 by default. Reach out if you would like to provision your SSO.",
-    icon: "zypsy-security-icon1.svg",
+    icon: CloudIcon,
   },
   {
     title: "Encryption at Rest & In-Transit",
     description:
       "Replay protects your data in transit with TLS v1.2 and strong ciphersuites. Replay protects your data at rest by encrypting it with AES-256 GCM.",
-    icon: "zypsy-security-icon2.svg",
+    icon: KeyIcon,
   },
   {
     title: "Intellectual property",
     description:
       "Replay is architected with your privacy in mind. We try our hardest not to collect sensitive information that we don't need, and will never sell your information.",
-    icon: "zypsy-security-icon1.svg",
+    icon: CloudIcon,
   },
   {
     title: "Private by default",
     description: "Replay will never view or analyze your replays without your explicit permission.",
-    icon: "zypsy-security-icon3.svg",
+    icon: ShieldIcon,
   },
-];
+] as const;
 
-export default function LoadingTip() {
-  const loadingPageTipIndex = useSelector(selectors.getLoadingPageTipIndex);
-  const dispatch = useDispatch();
-  const { title, description, icon } = TIPS[loadingPageTipIndex || 0];
-  const key = useRef<any>();
+export const LoadingTip: FC = () => {
+  const [tipIdx, setTipIdx] = useState(0);
```

Comment:
Is there some better naming we can use for `tipIdx`?

Reply:
`tipGripIdx` 😅 
jk, I'll come up with something.